addons/web_tour/static/src/scss/tip.scss

```diff
@@ -76,6 +76,32 @@
         &.right, &.left {
             animation-name: move-left-right !important;
         }
+        box-shadow: none;
+        border-color: transparent !important;
+        border-width: $o-tip-size / 2 !important;
+        background-color: transparent !important;
+        background-size: $o-tip-size * 2;
+        background-repeat: no-repeat;
+        background-position-x: 50%;
+        background-position-y: 50%;
+
```

Comment:
Using a SVG had two goals:
- Doing anything we want with a cool animation (we will have to ask a designer for that, the point is also removing the css animation and make it part of the svg) and it has to work when the tip is opened and closed (which is not the case here)
- Removing all the useless css: your commit is adding css so it's counterproductive

Let's pause this PR for now. I don't think we will have a viable SVG solution for 14.0 right now. I'll think about it tomorrow :wink: